collector/stat_activity.go

```diff
@@ -40,13 +40,19 @@ SELECT COALESCE(min(xact_start), current_timestamp) AS xact_start
   AND backend_xid IS NOT NULL /*postgres_exporter*/`
 
 	// Oldest backend timestamp
-	statActivityCollectorBackendStartQuery = `select min(backend_start) FROM pg_stat_activity /*postgres_exporter*/`
+	statActivityCollectorBackendStartQuery = `SELECT min(backend_start) FROM pg_stat_activity /*postgres_exporter*/`
+
+	// Oldest query in running state (long queries)"
+	statActivityCollectorActiveQuery = `
+SELECT EXTRACT(EPOCH FROM age(clock_timestamp(), min(query_start)))
+  FROM pg_stat_activity /*postgres_exporter*/`
```

Comment:
Needs a `state='active'` and at the same time, we should probably create a `statActivityCollectorOldestSnapshot` as
```sql
select age(clock_timestamp(), coalesce(min(query_start), clock_timestamp()))
from pg_stat_activity
where backend_xmin is not null
```